packages/manager/src/mocks/serverHandlers.ts

```diff
@@ -707,21 +712,32 @@ export const handlers = [
     if (request.headers.get('x-filter')) {
       const headers = JSON.parse(request.headers.get('x-filter') || '{}');
       const orFilters = headers['+or'];
+      const andFilters = headers['+and'];
 
-      if (orFilters) {
-        const filteredLinodes = linodes.filter((linode) => {
-          const filteredById = orFilters.some(
+      let filteredLinodes = linodes; // Default to the original linodes in case no filters are applied
```

Comment:
why we need changes in the mock??, we can try to explain in some comments here, since these mocks are owned by service owners, we need to explicit while making this changes. If this to make sure your changes is working, you can just attach the xFilter header request that should be fine